wmn-data.json

```diff
@@ -5470,25 +5490,26 @@
         "cat" : "finance"
        },
        {
-        "name" : "StreamLabs",
-        "uri_check" : "https://streamlabs.com/api/v6/user/{account}",
-        "uri_pretty" : "https://streamlabs.com/{account}/tip",
-        "e_code" : 200,
-        "e_string" : "display_name",
-        "m_code" : 401,
-        "m_string" : "UNAUTHORIZED",
-        "known" : ["veibae", "cutie_cori"],
-        "cat" : "finance"
-       },
-       {
-        "name" : "Stripchat",
-        "uri_check" : "https://stripchat.com/{account}",
-        "e_code" : 200,
-        "e_string" : "I Do in My Shows:",
-        "m_string" : "Oops. The page you were looking for doesn't exist",
-        "m_code" : 404,
-        "known" : ["DulcieRichard", "Katie-Mili"],
-        "cat" : "xx NSFW xx"
+        "name": "StreamLabs",
+        "uri_check": "https://streamlabs.com/api/v6/user/{account}",
+        "uri_pretty": "https://streamlabs.com/{account}/tip",
+        "e_code": 200,
+        "e_string": "\"id\":",
+        "m_code": 401,
+        "m_string": "Unauthorized",

```

Comment:
In the 401 response I see `\"message\":` which might be better `m_string`

Reply:
I don't see it, let's change it to `<title>Unauthorized</title>`.